src/forms/createPortalSchema.ts

```diff
@@ -4,9 +4,6 @@ import { PortalSchema } from "types/validation";
 
 import { spaceNameSchema } from "forms/spaceNameSchema";
 
-import { createUrlSchema } from "./factory/createUrlSchema";
-
 export const createPortalSchema = Yup.object().shape<PortalSchema>({
```

Comment:
I guess it can be removed entirely. We don't use `createPortalSchema` anymore.

Reply:
:fire: